feature_engine/timeseries/forecasting/base_forecast_transformers.py

```diff
@@ -81,6 +85,7 @@ def __init__(
         self.variables = _check_variables_input_value(variables)
         self.missing_values = missing_values
         self.drop_original = drop_original
+        self.group_by = _check_variables_input_value(group_by)
```

Comment:
if we defer the functionality to pandas, then we don't need this check. We just assign and let pandas handle the rest.

Reply:
That's right, pandas will handle it, Thanks for your help 🙏